### DISCUSSION TITLE
How to add "Extra" into Cloze card?

### DISCUSSION THREAD

**Discussion by GribbleP** [reuseman/flashcards-obsidian#110] [Q&A]
First of all, I LOVE this plugin. It has streamlined my studying and saved what will probably be YEARS of my life by the time I'm done with school / training. 

Question - is there a way to add info in the "extra" category for a Cloze deletion card? Right now, all the info that is written in line seems to only go into the "Text" box in Anki. I'd love to be able to throw in additional references into the extra box (like the attachment I tried to write into the line before generating the card) without giving away the answer by putting the reference in the "text" box. Attached some pics below to help if needed.

![image](https://user-images.githubusercontent.com/97628894/171050172-9df1654d-2e9e-4991-b7fa-df6c610a6931.png)

After generating card:
![image](https://user-images.githubusercontent.com/97628894/171050262-1baca04e-dd59-449e-8842-a733780d0774.png)

Card editing in Anki:
![image](https://user-images.githubusercontent.com/97628894/171050195-b0b56f73-a4dd-4aa5-a4ec-b5cc84c75b4d.png)

**Comment by apaiero** [comment]
This has also been something I've been trying to figure out - honestly, even without this it is an amazing plugin, but it would be a really great feature if anyone figured out a work-around for this.
